netrd/distance/netsimile.py

```diff
@@ -12,21 +12,22 @@
 """
 import networkx as nx
 import numpy as np
-import warnings
 from scipy.spatial.distance import canberra
 from scipy.stats import skew, kurtosis
 
 from .base import BaseDistance
-from ..utilities import ensure_undirected, ensure_unweighted
+from ..utilities import undirected, unweighted
 
 
 class NetSimile(BaseDistance):
     """Compares node signature distributions."""
 
+    @undirected
+    @unweighted
     def dist(self, G1, G2):
         """A scalable approach to network similarity.
 
-        A network similarity measure based on node signature distrubtions.
+        A network similarity measure based on node signature distributions.
```

Comment:
distrubtions was funny

Reply:
We put it off in the past, but given that this PR touches a ton of files we should just bite the bullet and do a general cleanup of the docs. I avoided modifying irrelevant files in this PR but there are still a bunch of typos in the docs, and some inconsistency in whether helper functions are documented.